js/private/js_helpers.bzl

```diff
@@ -177,8 +173,6 @@ def gather_runfiles(
     """
 
     # Includes sources
-    if type(sources) == "list":
```

Comment:
recommend to make it optional so caller isn't forced to construct an empty depset and can pass None instead

Reply:
I guess I would prefer that as well, just means changing the method signature 🤷